metals/src/main/scala/scala/meta/internal/builds/MavenBuildTool.scala

```diff
@@ -22,6 +22,7 @@ case class MavenBuildTool(userConfig: () => UserConfiguration)
   def bloopInstallArgs(workspace: AbsolutePath): List[String] = {
     def command(versionToUse: String) =
       List(
+        "generate-sources",
```

Comment:
I see that there is also `generate-test-sources` phase in [Lifecycle reference](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#lifecycle-reference). Probably we need to include it too.